### ISSUE TITLE
How many steps of the iteration can have acceptable results 

### ISSUE THREAD

**Issue by JK1532** [azraelkuan/tensorflow_wavenet_vocoder#1]
Thanks for sharing the great work.
I have trained on Chinese data for 48000 steps, but now I can only get noise through the mul-gengerate.py.And I found that the loss first decrease and then increase.Can you give me some advise.Thanks.

**Comment by azraelkuan** [comment]
sorry for late reply, my loss is about 1.5, and never increase, how about u?

**Comment by azraelkuan** [comment]
when i train on 4 1080Ti, only 10k, i can get the meaning of the sound, and at 40k, it will be much better
